packages/gasket-plugin-fastify/lib/index.js

```diff
@@ -42,6 +42,8 @@ module.exports = {
       // Enable middleware for fastify@3
       await app.register(middie);
 
+      await app.register(require('@fastify/middie'));
```

Comment:
I think you need to replace `middie` with `@fastify/middie`, correct?

See: https://www.npmjs.com/package/middie